bin/export-record-barcodes

```diff
@@ -57,6 +57,7 @@ def main():
         ScanProject(22475, "es", "irb"),
         ScanProject(22476, "ko", "irb"),
         ScanProject(22477, "vi", "irb"),
+        ScanProject(23089, "en", "irb_kiosk"),
```

Comment:
the purview should use a hyphen as a delimiter, not an underscore. See https://github.com/seattleflu/id3c-customizations/pull/118

Reply:
Thanks for that catch! I fixed it to match Jover's PR.

Reply:
Thank you!